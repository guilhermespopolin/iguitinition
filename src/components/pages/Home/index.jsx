import React from 'react'

import Text from 'components/ui/Text'
import Image from 'components/ui/Image'

import reactLogo from 'assets/imgs/react_logo.svg'

function Home() {
  return (
    <main css="height: 100vh;">
      <header
        css={`
          display: flex;
          align-items: center;
          justify-content: center;
          height: 40%;
          background-color: ${({ theme }) => theme.colors.black};
          color: ${({ theme }) => theme.colors.white};
        `}
      >
        <Image
          css={`
            & {
              transition: all 0.5s ease;
            }

            &:hover {
              transform: scale(1.1);
            }
          `}
          src={reactLogo}
          size="large"
        />
      </header>
      <Text css="padding: 0 0.5rem;" alignment="center">
        To get started, edit <code>src/components/pages/Home/index.jsx</code> and save to reload
      </Text>
    </main>
  )
}

export default Home
