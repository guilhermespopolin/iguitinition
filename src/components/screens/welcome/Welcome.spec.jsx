import React from 'react';
import renderer from 'react-test-renderer';

import Home from '.';

describe('<Home />', () => {
  const tree = renderer.create(<Home />).toJSON();

  it('should renders properly', () => {
    expect(tree).toMatchSnapshot();
  });
});
