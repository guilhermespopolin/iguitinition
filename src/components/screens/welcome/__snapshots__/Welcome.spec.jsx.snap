// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should renders properly 1`] = `
<div
  className="layout"
>
  <p>
    test-file-stub
  </p>
</div>
`;
